### PR TITLE
schannel: add CURLOPT_CERTINFO support

### PR DESCRIPTION
**Pull request by ajax16384**
allow to retrieve certinfo from winssl backend

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -578,6 +578,7 @@ if(NOT UNIX)
       if(HAVE_SCHANNEL_H)
         set(USE_SCHANNEL ON)
         set(SSL_ENABLED ON)
+	check_library_exists_concat("crypt32" CertFreeCertificateContext HAVE_LIBCRYPT32)
```
Comment (mback2k) [inline CMakeLists.txt]: Please fix the indentation by adding one additional space in front of this line.

File: lib/vtls/schannel.c [hunk 2]
```diff
@@ -694,6 +696,30 @@ schannel_connect_step3(struct connectdata *conn, int sockindex)
...
+
+    if((sspi_status != SEC_E_OK) || (ccert_context == NULL)) {
+      failf(data, "schannel: failed to retrieve remote cert context");
+      return CURLE_SSL_CONNECT_ERROR;
+    }
+
+    result = Curl_ssl_init_certinfo(data, 1);
+    if(!result) {
+      if(((ccert_context->dwCertEncodingType & X509_ASN_ENCODING) != 0) &&
+        (ccert_context->cbCertEncoded > 0)) {
```
Comment (mback2k) [inline lib/vtls/schannel.c]: Please fix the indentation by adding one additional space in front of this line.

**Comment by mback2k** [comment]
The proposed patch / merge request looks good to me, aside from the small indentation fixes. @bagder do you have any final comments before merging?

**Comment by jay** [comment]
@mback2k hold off there's a patch release pending that's why no features have been added

**Comment by mback2k** [comment]
@jay Yep, I saw that information. :-)

File: CMakeLists.txt (hunk 1 above)
Comment (ajax16384) [inline CMakeLists.txt]: ok

File: lib/vtls/schannel.c (hunk 2 above)
Comment (ajax16384) [inline lib/vtls/schannel.c]: ok

**Comment by bagder** [comment]
my comment: :+1:

**Comment by bagder** [comment]
thanks!

**Pull request closed without merging**